frontend/src/prompted_readings/analysis_view.js

```diff
@@ -26,17 +26,23 @@ class AnalysisView extends React.Component {
     }
 
     render() {
+
         if (this.state.analysis !== null) {
             const {  // object destructuring:
                 total_view_time,
+                run_mean_reading_analysis_for_questions,
                 compute_median_view_time,
             } = this.state.analysis;
-
+            //console.log(run_mean_reading_analysis_for_questions)
             return (
                 <div>
                     <h1>Analysis of Student Responses</h1>
                     <h3>Total view time</h3>
                     <p>{total_view_time} seconds</p>
+                    <h3>Mean Reading Time for Questions</h3>
```

Comment:
On the basis of my comment about the output of `mean_reading_time_for_a_question`, this will probably have to change.